docs/quality-principles/introduction.md

```diff
@@ -0,0 +1,44 @@
+# Dasharo Quality Principles
+
+## Secure Delivery
+
+Each Dasharo release is secured cryptographically, ensuring with RSA 4096 keys,
```

Comment:
ensuring with **RSA-4096** keys **the** authenticity of binary SHA-256 integrity checksum

Reply:
applied grammarly fixes